AutoMatrix/AutoMatrix.py

```diff
@@ -454,24 +454,27 @@ def ProcessVolume(self)-> None:
         '''
         Function that will apply the matrix to all the files
         '''
-
+        volume_storage_node = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLVolumeArchetypeStorageNode")
+        model_storage_node = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLModelStorageNode")
```

Comment:
Probably shouldn't add these to the scene. Or, if you do, remove them when you're done.